pentasharp/Data/AppDbContext.cs

```diff
@@ -0,0 +1,19 @@
+using Microsoft.EntityFrameworkCore;
+
+namespace pentasharp.Data 
+{
+    public class AppDbContext : DbContext
+    {
+        public AppDbContext(DbContextOptions<AppDbContext> options) : base(options)
+        {
+        }
+

```

Comment:
remove extra lines

Reply:
you should leave one line as space between AppDbContext, not more than one or less